tests/conftest.py

```diff
@@ -6,6 +6,6 @@
 from agoradatatools.etl import utils
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
May I ask why you are using `autouse` here?

Reply:
removed

Reply:
removed in latest commit. I was playing around with it and forgot to remove it before committing earlier